localstack/cli/localstack.py

```diff
@@ -37,17 +47,38 @@ def _setup_cli_debug():
     setup_logging()
 
 
+def _publish_cmd_as_analytics_event(cmd: str):
+    event = Event(
+        name="cli_cmd",
+        payload={"cmd": cmd},
+        metadata=EventMetadata(
+            session_id=get_session_id(),
+            client_time=str(datetime.datetime.now()),  # TODO: consider using utcnow()
```

Comment:
I elected not to use `utcnow()` here to keep it in line with how we're doing `client_time` elsewhere, but this is problematic IMO due to timezones.

Reply:
good catch, definitely something we should clean up and unify